### PR TITLE
Example for MultiMap [API-478]

### PR DESCRIPTION
**Pull request by utku-caglayan**

### CODE REVIEW

File: examples/multimap/main.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	m, err := client.GetMultiMap(ctx, mapName)
+	if err != nil {
+		log.Fatal(err)
+	}
+	// Populate map
+	if success, err := m.Put(ctx, "key", "value1"); err != nil {
+		log.Fatal(err)
+	} else if !success {
+		log.Fatal("multi-map put operation failed")
+	}
```
Comment (yuce) [inline examples/multimap/main.go:32]: Let's prefer to not use `else`.

File: examples/multimap/main.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	if success, err := m.Put(ctx, "key", "value2"); err != nil {
+		log.Fatal(err)
+	} else if !success {
+		log.Fatal("multi-map put operation failed")
+	}
```
Comment (yuce) [inline examples/multimap/main.go:37]: Let's prefer to not use `else`.

File: examples/multimap/main.go [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+		log.Fatal(err)
+	}
+	// Get a random map
+	rand.Seed(time.Now().Unix())
+	mapName := fmt.Sprintf("sample-%d", rand.Int())
+	m, err := client.GetMultiMap(ctx, mapName)
+	if err != nil {
+		log.Fatal(err)
+	}
+	// Populate map
```
Comment (yuce) [inline examples/multimap/main.go:27]: `Populate the map.`

File: examples/multimap/main.go [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+		log.Fatal("multi-map put operation failed")
+	}
+
+	// Get both values under the same key
+	values, err := m.Get(ctx, "key")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// We see values in an interface slice
```
Comment (yuce) [inline examples/multimap/main.go:45]: How about something like: `An interface slice contains the values`.

**Review by yuce — APPROVED** [review]
Looks good! Just a few things to make it even better.

File: examples/multimap/main.go (hunk 3 above)
Comment (utku-caglayan) [reply examples/multimap/main.go]: Sure also changes map example

**Review by emreyigit — APPROVED** [review]

**Pull request merged**